src/main/java/ui/UI.java

```diff
@@ -431,6 +432,9 @@ public MenuControl getMenuControl() {
     }
 
     public void setDefaultWidth() {
+        if (isTestMode()) {
+            triggerEvent(new WindowResizeEvent(WindowResizeEvent.EventType.DEFAULT_SIZE_WINDOW));
+        }
```

Comment:
let's try to minimize this kind of code. It reduces the credibility of tests when the code behaves differently in test mode.